lib/internal/process.js

```diff
@@ -60,7 +60,7 @@ function setupConfig(_source) {
       .replace(/"/g, '\\"')
       .replace(/'/g, '"');
 
-  process.config = JSON.parse(config, function(key, value) {
+  process.config = exports.config = JSON.parse(config, (key, value) => {
```

Comment:
This line doesn't make a copy, `process.config` and `exports.config` point to the same object.